index.html

```diff
@@ -0,0 +1,429 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <title>Basic HTML Page Structure</title>
+    <link rel="stylesheet" href="src/css/style.css" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
```

Comment:
It's better to use this script tag on the bottom of the body of your html content, don't use it here in head tag